### PR TITLE
Add make target for fetching base images of custom test images we build

### PR DESCRIPTION
**Pull request by happz**
Very useful for refreshing test images.

Pull Request Checklist

* [x] implement the feature

### CODE REVIEW

**Review by martinhoyer — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -117,6 +118,9 @@ TMT_TEST_IMAGES_TARGETS := $(foreach image,$(TMT_TEST_CONTAINER_IMAGES),images-t
 images-tests: $(TMT_TEST_IMAGES_TARGETS)  ## Build customized images for tests
 	podman images | grep 'localhost/$(TMT_TEST_CONTAINER_IMAGE_NAME_PREFIX)/' | sort
 
+images-tests-bases:  ## Download base images for custom test images
+	for image in $(TMT_TEST_IMAGE_BASES); do podman pull "$$image"; done
```
Comment (thrix) [inline Makefile:122]: pull supports multiple images

```suggestion
	podman pull $(TMT_TEST_IMAGE_BASES)
```

**Review by thrix — APPROVED** [review]
LGTM, just a small improvement

File: Makefile (hunk 1 above)
Comment (happz) [reply Makefile]: Nice, addressed in d5c933e4.

**Review by therazix — APPROVED** [review]

**Comment by happz** [comment]
Trivial change not affecting tests, merging right away.

**Pull request merged**